pkg/arvo/app/wallet.hoon

```diff
@@ -11,107 +11,107 @@
-  ==
-/=  worker
-  /^  octs
-  /;  as-octs:mimes:html
-  /:  /===/app/bitcoin/js/worker
-  /|  /js/
-      /~  ~
-  ==
-/=  bledger
-  /^  octs
-  /;  as-octs:mimes:html
-  /:  /===/app/bitcoin/js/bledger
-  /|  /js/
-      /~  ~
-  ==
-/=  bmanager
-  /^  octs
-  /;  as-octs:mimes:html
-  /:  /===/app/bitcoin/js/bmanager
-  /|  /js/
-      /~  ~
-  ==
-/=  bpath
-  /^  octs
-  /;  as-octs:mimes:html
-  /:  /===/app/bitcoin/js/bpath
-  /|  /js/
-      /~  ~
-  ==
-/=  binput
-  /^  octs
-  /;  as-octs:mimes:html
-  /:  /===/app/bitcoin/js/binput
-  /|  /js/
-      /~  ~
-  ==
-/=  bhelper
-  /^  octs
-  /;  as-octs:mimes:html
-  /:  /===/app/bitcoin/js/bhelper
-  /|  /js/
-      /~  ~
-  ==
-/=  bcommon
-  /^  octs
-  /;  as-octs:mimes:html
-  /:  /===/app/bitcoin/js/bcommon
-  /|  /js/
-      /~  ~
-  ==
-/=  style
-  /^  octs
-  /;  as-octs:mimes:html
-  /:  /===/app/bitcoin/css/index
-  /|  /css/
-      /~  ~
-  ==
-/=  bitcoin-png
-  /^  (map knot @)
-  /:  /===/app/bitcoin/img  /_  /png/
+:: /=  index
+::   /^  octs
+::   /;  as-octs:mimes:html
+::   /:  /===/app/bitcoin/index
+::   /|  /html/
+::       /~  ~
+::   ==
+:: /=  tile-js
+::   /^  octs
+::   /;  as-octs:mimes:html
+::   /:  /===/app/bitcoin/js/tile
+::   /|  /js/
+::       /~  ~
+::   ==
+:: /=  script
+::   /^  octs
+::   /;  as-octs:mimes:html
+::   /:  /===/app/bitcoin/js/index
+::   /|  /js/
+::       /~  ~
+::   ==
+:: /=  bcoin
+::   /^  octs
+::   /;  as-octs:mimes:html
+::   /:  /===/app/bitcoin/js/bcoin
+::   /|  /js/
+::       /~  ~
+::   ==
+:: /=  proxy
+::   /^  octs
+::   /;  as-octs:mimes:html
+::   /:  /===/app/bitcoin/js/proxy
+::   /|  /js/
+::       /~  ~
+::   ==
+:: /=  logger
+::   /^  octs
+::   /;  as-octs:mimes:html
+::   /:  /===/app/bitcoin/js/logger
+::   /|  /js/
+::       /~  ~
+::   ==
+:: /=  worker
+::   /^  octs
+::   /;  as-octs:mimes:html
+::   /:  /===/app/bitcoin/js/worker
+::   /|  /js/
+::       /~  ~
+::   ==
+:: /=  bledger
+::   /^  octs
+::   /;  as-octs:mimes:html
+::   /:  /===/app/bitcoin/js/bledger
+::   /|  /js/
+::       /~  ~
+::   ==
+:: /=  bmanager
+::   /^  octs
+::   /;  as-octs:mimes:html
+::   /:  /===/app/bitcoin/js/bmanager
+::   /|  /js/
+::       /~  ~
+::   ==
+:: /=  bpath
+::   /^  octs
+::   /;  as-octs:mimes:html
+::   /:  /===/app/bitcoin/js/bpath
+::   /|  /js/
+::       /~  ~
+::   ==
+:: /=  binput
+::   /^  octs
+::   /;  as-octs:mimes:html
+::   /:  /===/app/bitcoin/js/binput
+::   /|  /js/
+::       /~  ~
+::   ==
+:: /=  bhelper
+::   /^  octs
+::   /;  as-octs:mimes:html
+::   /:  /===/app/bitcoin/js/bhelper
+::   /|  /js/
+::       /~  ~
+::   ==
+:: /=  bcommon
+::   /^  octs
+::   /;  as-octs:mimes:html
+::   /:  /===/app/bitcoin/js/bcommon
+::   /|  /js/
+::       /~  ~
+::   ==
+:: /=  style
+::   /^  octs
+::   /;  as-octs:mimes:html
+::   /:  /===/app/bitcoin/css/index
+::   /|  /css/
+::       /~  ~
+::   ==
+:: /=  bitcoin-png
+::   /^  (map knot @)
+::   /:  /===/app/bitcoin/img  /_  /png/
```

Comment:
Note: these can all be removed, not just commented out.

Reply:
whoops, I had collapsed all these comments and forgot to delete them. thanks!